Airports/EGSS/RMA.txt

```diff
@@ -1,32 +1,21 @@
-Luton Westerly Buffer     S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
-                          N052.01.01.405 W000.02.28.960 N052.01.52.172 E000.02.28.960
-                          N052.01.52.172 E000.02.28.960 N051.50.43.759 E000.02.28.960
-                          N051.50.43.759 E000.02.28.960 N051.50.20.324 W000.02.28.960
-                          N051.50.20.324 W000.02.28.960 N052.01.01.405 W000.02.28.960
-Luton Easterly Buffer     S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
-                          N052.01.01.405 W000.02.28.960 N052.01.52.172 E000.02.28.960
-                          N052.01.52.172 E000.02.28.960 N051.52.40.970 E000.02.28.960
-                          N051.52.40.970 E000.02.28.960 N051.50.28.167 W000.00.49.258
-                          N051.50.28.167 W000.00.49.258 N051.50.04.180 W000.05.54.180
-                          N051.50.04.180 W000.05.54.180 N051.49.35.890 W000.08.41.569
-                          N051.49.35.890 W000.08.41.569 N051.53.46.478 W000.02.28.960
-                          N051.53.46.478 W000.02.28.960 N052.01.01.405 W000.02.28.960
-Stansted RMA              S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
-                          N052.11.04.000 W000.21.59.000 N052.11.04.000 E000.05.28.000
-                          N052.11.04.000 E000.05.28.000 N052.00.45.000 E000.47.24.000
-                          N052.00.45.000 E000.47.24.000 N051.54.21.745 E000.48.40.101
-                          N051.54.21.745 E000.48.40.101 N051.53.17.926 E000.35.08.869
-                          N051.53.17.926 E000.35.08.869 N051.46.59.000 E000.36.29.000
-                          N051.46.59.000 E000.36.29.000 N051.45.08.000 E000.13.09.000
-                          N051.45.08.000 E000.13.09.000 N051.40.55.000 E000.06.52.000
-                          N051.40.54.998 E000.06.51.998 N051.41.19.806 E000.05.09.427
-                          N051.41.19.806 E000.05.09.427 N051.41.51.359 E000.03.31.753
-                          N051.41.51.359 E000.03.31.753 N051.42.28.046 E000.01.58.906
-                          N051.42.28.046 E000.01.58.906 N051.43.09.589 E000.00.31.592
-                          N051.43.09.589 E000.00.31.592 N051.43.55.670 W000.00.49.523
-                          N051.43.55.670 W000.00.49.523 N051.44.45.940 W000.02.03.823 
-                          N051.44.45.940 W000.02.03.823 N051.45.49.997 W000.03.15.998
-                          N051.45.50.000 W000.03.16.000 N051.51.01.000 W000.00.25.000
-                          N051.51.01.000 W000.00.25.000 N051.50.36.000 W000.02.52.000
-                          N051.50.36.000 W000.02.52.000 N051.59.35.000 W000.10.56.000
-                          N051.59.35.000 W000.10.56.000 N052.11.04.000 W000.21.59.000
\ No newline at end of file
+Stansted RMA               S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```

Comment:
Your picture for the STAR data is missing the boundary between F and G.

Reply:
Updated